pkg/grpc/server.go

```diff
@@ -28,6 +34,42 @@ func init() {
 			util.DecimalExponentialBuckets(-3, 6, 2)))
 }
 
+// ServerRMDHandler is meant to wrap ocgrpc.ServerHandler, and exports additional information
```

Comment:
Apart from some abbreviations that are obvious based on the context (HTTP, CAS, AC), the Buildbarn codebase doesn't use abbreviations a lot. Would it make sense to call this requestMetadataStatsHandler?

Also some remarks related to this code:

- The Buildbarn codebase often makes types private and uses constructor-like functions: `func NewFooBlobAccess() BlobAccess`. This has the advantage that newly introduced parameters (i.e., new members of ServerRMDHandler that need initialization) will cause explicit build failures at the place of instantiation. It also means we don't need to redundantly document methods (in this case what you needed to do with TagRPC). Could we do this as well?
- Could you place this type in a separate source file? One of the reasons we do this in Buildbarn is because it makes it easier to see which code has (some) unit testing coverage and which code does not. Simply run `ls` and see which source files have an accompanying `_test.go`. On that note: could you add some unit tests?